package/opt/hassbian/suites/cloud9.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+function cloud9-show-short-info {
+  printf "Cloud9 install script for Hassbian.\\n"
+}
+
+function cloud9-show-long-info {
+  printf "Installs Cloud9 IDE onto this system.\\n"
+  printf "Cloud9 IDE is an online integrated development environment. It supports hundreds of programming languages.\\n"
+}
+
+function cloud9-show-copyright-info {
+  printf "This script was originally contributed by Ludeeus <https://github.com/ludeeus>.\\n"
+}
+
+function cloud9-install-package {
+node=$(which node)
+if [ -z "${node}" ]; then #Installing NodeJS if not already installed.
+  printf "Downloading and installing NodeJS...\\n"
+  curl -sL https://deb.nodesource.com/setup_8.x | bash -
+  apt install -y nodejs
+fi
+
+echo "Changing to the homeassistant user"
+sudo -u homeassistant -H /bin/bash << EOF
+  printf "Downloading and installing Cloud9 SDK...\\n"
+  git clone git://github.com/c9/core.git /home/homeassistant/c9sdk
```

Comment:
Wouldn't it be preferable to have the installation in a proper location instead of in a users home directory?
Example could be AppDaemon.

Reply:
I will look into moving it to something like /etc/

Reply:
More like `/opt`